tests/ecall_ocall/enc/helpers.h

```diff
@@ -5,23 +5,26 @@
 
 // TLS wrapper for a bit of syntactic sugar, in absence of thread_local
 // support.
-struct TLSWrapper
+class tls_wrapper
```

Comment:
This test was written before we had thread-local support. Now that we have thread-local support, it is better to lock that down. So we can use __thread or thread_local instead of oe_thread_key_create etc.